src/Application/Channels/Commands/PatchChannelCommand.cs

```diff
@@ -13,7 +13,7 @@ namespace Hippo.Application.Channels.Commands;
 public class PatchChannelCommand : IRequest
 {
     public Guid ChannelId { get; set; }
-    public Field<List<UpdateEnvironmentVariableDto>?> EnvironmentVariables { get; set; } = null!;
+    public Field<List<UpdateEnvironmentVariableDto>> EnvironmentVariables { get; set; } = null!;
```

Comment:
I am not sure if this will still allow the patch command to receive null values from the front-end, will test

Reply:
Oh! I completely forgot why we’d want to accept null values. That makes sense. Yes this change will prevent us from accepting null values. Will do some changes to accommodate that.

Reply:
I've tried passing EnvironmentVariables as null and it worked, even when the list was not nullable, I think it's because of the null-forgiving operator (https://docs.microsoft.com/en-us/dotnet/csharp/language-reference/operators/null-forgiving), but maybe keeping the List as nullable would be more indicative of the values it can receive?

Reply:
ACK